mirascope/prompts.py

```diff
@@ -151,3 +171,16 @@ def messages_fn(self) -> list[tuple[str, str]]:
 
     setattr(cls, "messages", property(messages_fn))
     return cls
+
+
+def tags(args: Union[list[str], str]) -> Callable[[Type[T]], Type[T]]:
+    def tags_fn(model_class: Type[T]) -> Type[T]:
+        # Step 2: Attach the tags to the model class
```

Comment:
what is step 1?

Reply:
Oh oops, I had other comment, I will update